packages/web-client/app/components/card-pay/header/workflow-tracker/dropdown-list/index.ts

```diff
@@ -3,15 +3,62 @@ import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
 
 import WorkflowPersistence from '@cardstack/web-client/services/workflow-persistence';
+import { tracked } from '@glimmer/tracking';
 
 interface CardPayHeaderWorkflowTrackerDropdownListArgs {}
 
 export default class CardPayHeaderWorkflowTrackerDropdownList extends Component<CardPayHeaderWorkflowTrackerDropdownListArgs> {
   @service declare workflowPersistence: WorkflowPersistence;
+  element!: HTMLElement;
+  initialPosition?: number;
+  @tracked currentPosition?: number;
+
+  @action setupEventListeners(element: HTMLElement) {
+    this.element = element;
+
+    let item =
+      document.getElementById('workflow-tracker-active-list')
+        ?.firstElementChild ??
+      document.getElementById('workflow-tracker-completed-list')
+        ?.firstElementChild;
+    this.initialPosition = item?.getBoundingClientRect().top;
+
+    element.addEventListener('scroll', () => {
+      this.currentPosition = item?.getBoundingClientRect().top;
+    });
+
+    element.onfocus = this.lockBodyScroll;
+    element.onblur = this.unlockBodyScroll;
+    element.onmouseover = this.lockBodyScroll;
+    element.onmouseout = this.unlockBodyScroll;
+  }
+
+  get isScrolled() {
+    return Boolean(
+      this.currentPosition && this.currentPosition !== this.initialPosition
+    );
+  }
```

Comment:
isScrolled can be determined via `this.element.scrollTop > 0` and set in the event listener, so we don't need to use the children's positions to determine whether the contents of an element are scrolled or not:

https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTop

Reply:
thanks!

Reply:
> `isScrolled` can be determined via `this.element.scrollTop > 0` and set in the event listener, so we don't need to use the children's positions to determine whether the contents of an element are scrolled or not:
> 
> https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTop

However, this doesn't tell me which list is scrolling.